### PR TITLE
Update instructions

### PR DESCRIPTION
**Pull request by IvhaniMaselesele**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
+simple programming assignment.
+
+Instructions
+------------
+This assignment requires you to write a small application that reads the English dictionary 
+and **calculates the number of anagrams that exist for each word length**. 
+
+
+Your code should:
+ - make use of the included file Dictionary.txt
```
[the remark this answers is not in the document]
Comment (IvhaniMaselesele) [reply README.md:14]: Fair point. However, I think this is acceptable punctuation for bullet points.

See example A: https://www.purchase.edu/editorial-style-guide/general-style-preferences/punctuation/bulleted-and-numbered-lists/#:~:text=Preferred%20style%20is%20without%20terminal,is%20maintained%20within%20a%20document.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+
+Your code should:
+ - make use of the included file Dictionary.txt 
+ - cater for dictionaries with words of different lengths
+ - write your results to the console in the form "Words with the character length of x had y anagrams"
+ - include the total time in milliseconds somewhere in your console output
+ - not write to the console on processing each word
+
+NB: 
+ - The definition of an anagram can be found here: https://www.google.com/search?q=definition+of+anagram
```
[the remark this answers is not in the document]
Comment (IvhaniMaselesele) [reply README.md:21]: Any specific reason why you prefer one source?

I think its better to present them with everything. They can choose one or multiple of these. In general, I feel this helps get a better understanding of something.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+ - Anagram Service class has been provided to help you structure your code.
+ - Feel free to change the structure of the solution if you feel it helps optimise execution speed, memory usage etc.
+ - Feel free to use Google when crafting your solution.
+ - Document assumptions (if any) as comments in code.
+ - This assignment only requires you to calculate anagrams that are **contained** in the provided dictionary.
+
+
+Next Steps:
+ - Please re-zip your solution **excluding** all unnecessary files (e.g. *.user, *.suo, bin folder, obj folder)
```
[the remark this answers is not in the document]
Comment (IvhaniMaselesele) [reply README.md:30]: Fair point.

I believe this was added when we still received submissions via email. Emails would block compiled files.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (liampulles) [reply README.md:14]: Ok cool

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (liampulles) [reply README.md:21]: At the moment we are essentially telling them that if they are unsure of what an anagram definition is, they should google it - is it not redundant?
Its okay with me though

**Pull request merged**